src/test/unittest/ut.c

```diff
@@ -466,3 +467,22 @@ ut_err(const char *file, int line, const char *func,
 
 	errno = saveerrno;
 }
+
+/*
+ * ut_anon_mmap -- mmap of anonymous memory
+ */
+void *
+ut_anon_mmap(const char *file, int line, const char *func, int len)
+{
+        void *base;
+
+        void *dest = util_map_hint(len);
```

Comment:
I agree about introducing the dependency. I originally had it coded so it didn't use this. The reason I use the util_map_hint() is that I wanted the mmap'd region to be controllable in terms of less than or > than the pm file mapping, which is controlled by pmem_map and the process . I tried to do this with mmap, providing the first parameter with an address I wanted it to use, and it failed.